src/TgSharp.TL/TL/Account/TLContentSettings.cs

```diff
@@ -26,7 +26,10 @@ public override int Constructor
 
         public void ComputeFlags()
         {
-            // do nothing
+            Flags = 0;
+            Flags = SensitiveEnabled ? (Flags | 1) : (Flags & ~1);
+            Flags = SensitiveCanChange ? (Flags | 2) : (Flags & ~2);
```

Comment:
can separate the flags fix in a different commit pls?

Reply:
it's separated now

Reply:
so does the PR work if you separate this commit to a new PR?

Reply:
yeah removed it from the PR